src/backend/routers/para.test.ts

```diff
@@ -44,3 +77,52 @@ test("createPara", async (t) => {
       .some((mail) => mail.subject?.includes("confirmation"))
   );
 });
+
+test("duplicateEmail", async (t) => {
+  const { db } = await getTestServer(t, { authenticateAs: "para" });
+
+  await t.throwsAsync(() => {
+    return db
+      .insertInto("user")
+      .values({
+        first_name: "Foos",
+        last_name: "Bar",
+        email: "john@example.com",
+        role: "staff",
+      })
+      .execute();
+  });
+});
+
+test("assignParaToCaseManager", async (t) => {
+  const { trpc, seed } = await getTestServer(t, { authenticateAs: "para" });
+
+  await trpc.para.assignParaToCaseManager.mutate({
+    para_id: seed.paraJohn.user_id,
+  });
+
+  const myParas = await trpc.para.getMyParas.query();
+  t.is(myParas.length, 1);
```

Comment:
an assertion before `.assignParaToCaseManager()` would be good to confirm that this is initially `0` (often useful to make sure seed data isn't affecting tests)